ocs_ci/utility/workloads/jenkins.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module implements all the functionalities required for setting up and
+running "Jenkins" like workloads on the pods.
+
+This module implements few functions
+setup(): for setting up git utility on the pod and any necessary
+    environmental params.
+run(): for running 'git clone' on pod to simulate a working environment for
+    a developer
+
+Note: The above mentioned functions will be invoked from Workload.setup()
+and Workload.run() methods along with user provided parameters.
+"""
+import logging
+
+from ocs_ci.ocs import exceptions
+
+
+log = logging.getLogger(__name__)
+
+DISTROS = {"Debian": "apt-get", "RHEL": "yum"}
+
+
+def find_distro(io_pod):
```

Comment:
I think we already have a function for this, I can dig, but should be part of utils.

Reply:
Indeed - ocs_ci.utility.workloads.fio.find_distro

Reply:
Thinking maybe the different workload should be represented in classes that inherit from a base one with the common methods, like find_distro

Reply:
created helpers.py under workloads directory